.github/actions/send_slack_notifications/action.yml

```diff
@@ -0,0 +1,38 @@
+name: send_slack_notifications
+description: Send Slack notifications
+inputs:
+  SLACK_WEBHOOK_URL:
+    description: 'Slack webhook URL'
+    required: true
+  status:
+    description: 'Job status'
+    required: true
+  release_type:
+    description: 'Release type'
+    required: true
+  version:
+    description: 'Version'
+    required: true
+    default: 'N/A'
+runs:
+  using: composite
+  steps:
+  - name: Send Slack Notification on Success
+    if: inputs.status == 'success'
+    run: |-
+        curl -X POST -H 'Content-type: application/json' \
```

Comment:
since the slack webhook generated using old approach, safest option was to send the message using curl, until we update the workflow to use latest approach then we can make it parametric and use third party slack-action.